eth-bytecode-db/eth-bytecode-db-proto/proto/v2/eth-bytecode-db.proto

```diff
@@ -101,6 +101,9 @@ message Source {
 
   /// Info about the runtime code (sourcemaps, linkreferences, immutables) encoded as a json.
   optional string deployed_bytecode_artifacts = 12;
+
+  /// If true, the verified contract was a ERC-5202 compatible blueprint contract
+  bool is_blueprint = 13;
```

Comment:
If we gonna have multiple similar customizations, maybe it's better to introduce a single field with such "tags". Can be a string/enum or a list of strings/enums. Wdyt?

Reply:
I like the idea of having a list of enums in case we have similar customizations in the future. However, I'm not sure it is a good thing to add right now. For now I see no more tags which could potentially be added in the future, and it may be that the schema with list of strings/enums will not fit when such tags appear. 

E.g., I can imagine the situation where the contract may be tagged as a proxy with some additional info that should be returned along with the tag

So, imo it may be better to implement the simplest approach until an actual use case appear